pkg/util/dockerutils.go

```
@@ -219,3 +219,30 @@ func GetContainerHealth(container docker.APIContainers) string {
 	}
 	return match
 }
+
+// GetAppContainers retrieves docker containers for a given sitename.
+func GetAppContainers(sitename string) []docker.APIContainers {
+	label := map[string]string{"com.ddev.site-name": sitename}
+	sites, err := FindContainersByLabels(label)
+	if err != nil {
+		log.Fatal("failed to retrieve containers for ", sitename, err)
+	}
+	return sites
+}
+
+// GetContainerEnv returns the value of a given environment variable from a given container.
+func GetContainerEnv(key string, container docker.APIContainers) string {
+	client := GetDockerClient()
+	inspect, err := client.InspectContainer(container.ID)
+	if err != nil {
+		log.Debug("failed to retreive container ", ContainerName(container))
```

Seems like we want to return an error as well?